tss-esapi/src/context/tpm_commands/asymmetric_primitives.rs

```diff
@@ -75,8 +75,63 @@ impl Context {
         }
     }
 
-    // Missing function: ECDH_KeyGen
-    // Missing function: ECDH_ZGen
+    /// Generate an ephemeral key pair.
+    pub fn ecdh_keygen(
```

Comment:
I think I would prefer it to be ```ecdh_key_gen``` that would be more in line with how the other methods are named.

Reply:
Understood! :+1: